service/src/main/java/uk/gov/hmcts/reform/payment/model/FeeDto.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.payment.model;
```

Comment:
Hmm not sure if it's worth having both "payment" and "fnp" packages? How different they are?